core/services/ocrbootstrap/delegate.go

```diff
@@ -129,6 +130,35 @@ func (d *Delegate) ServicesForSpec(jobSpec job.Job) (services []job.ServiceCtx,
 	return []job.ServiceCtx{configProvider, job.NewServiceAdapter(bootstrapper)}, nil
 }
 
+// OnCreateJob satisfies the job.Delegate interface.
+func (d *Delegate) OnCreateJob(jb job.Job, q pg.Queryer) error {
+	spec := jb.BootstrapSpec
+	if spec == nil {
+		return errors.Errorf("Bootstrap.Delegate expects an *job.BootstrapSpec to be present, got %v", spec)
+	}
+
+	relayer, exists := d.relayers[spec.Relay]
+	if !exists {
+		return errors.Errorf("%s relay does not exist is it enabled?", spec.Relay)
+	}
+
+	rargs := types.RelayArgs{
+		JobID:       spec.ID,
+		ContractID:  spec.ContractID,
+		New:         true,
```

Comment:
I am confused about why job details are leaking in to the relayer API here with `New` :confused: 
Pre-existing, but seems to be leading us down the wrong path.

Reply:
Not sure why it's a member of the struct, but this is unused/ignored in this particular case.  We could just leave off the `New:` field as it doesn't matter whether it's true or false, just included it for completeness.